test/gtest/ucs/test_memtype_cache.cc

```diff
@@ -32,7 +32,25 @@ class test_memtype_cache : public ucs::test_with_param<ucs_memory_type_t> {
         ucs::test_with_param<ucs_memory_type_t>::cleanup();
     }
 
-    void test_lookup_found(void *ptr, size_t size,
+    void check_lookup(const void *ptr, size_t size,
+                      ucs_status_t status,
+                      ucs_memory_type_t mem_type,
+                      bool found, 
```

Comment:
found means if you expect it to be found, or if it was actually found?
pls rename accordingly

Reply:
it means this:

> found means if you expect it to be found

renamed to `expect_found`